src/app/modules/ix-date/ix-date.component.ts

```diff
@@ -1,18 +1,26 @@
 import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
-import { utcToZonedTime } from 'date-fns-tz';
+import { utcToZonedTime, zonedTimeToUtc } from 'date-fns-tz';
 
 @Component({
-  templateUrl: './ix-date.component.html',
   selector: 'ix-date',
+  templateUrl: './ix-date.component.html',
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class IxDateComponent {
   @Input() date: number | Date;
-  @Input() dateFormat: string = null;
-  @Input() timeFormat: string = null;
-  @Input() machineTimezone: string = Intl.DateTimeFormat().resolvedOptions().timeZone;
+  @Input() appliedTimezone: string = Intl.DateTimeFormat().resolvedOptions().timeZone;
+  @Input() machineTimezone: string;
```

Comment:
I would try to read machine zone in this component, because otherwise you'd need to be passing it in in every place where ix-date is used.